### PR TITLE
TASK SIO-4928 set GPT as default partition, update OSes

### PR DESCRIPTION
**Pull request by avolovplesk**

### CODE REVIEW

**Comment by redikultsevsilver** [comment]
Maybe it worth changing cloud_id as well? https://github.com/solusio/solus-cloud-images/blob/public/scripts/plesk/install_plesk.sh#L158 
I doubt it's used anywhere at the moment, therefore adding some consistency to that file as well should not ruin anything in my opinion.

**Comment by redikultsevsilver** [comment]
What about changing the `output_image_name` for the windows builds, e.g. in windows/windows-2019-plesk.json, windows/windows-2019.json, windows/windows-2022-plesk.json, windows/windows-2022.json?

Also, I see that while you do rename the output images, from solus to solusvm, you do not rename the files themselves and the files are still called like `centos/solus-centos-8-stream.json`. Is is intentional?

**Comment by avolovplesk** [comment]
Added changes according to suggestions from redikultsevsilver

**Review by redikultsevsilver — APPROVED** [review]
Do not forget to preserve the links on https://images.dev.solusvm.com/ and https://images.prod.solus.io/ :) Or change the links in the product and leave 301 redirect to the new links on the old links.

File: centos/solusvm2-centos-8-stream.json [hunk 1]
```diff
@@ -66,4 +66,4 @@
       ]
     }
   ]
-}
+}
```
Comment (oneumyvakin) [inline centos/solusvm2-centos-8-stream.json]: Please add a new line to the end of the file.

File: debian/cleanup.sh [hunk 2]
```diff
@@ -38,6 +38,9 @@ clean_garbage()
 
 	# Clean evidences of your activity
 	echo -n > /root/.bash_history
+
+	# Clean /etc/resolv.conf
+  echo -n > /etc/resolv.conf
```
Comment (oneumyvakin) [inline debian/cleanup.sh]: Please add indents.

File: fedora/solusvm2-fedora-29.json [hunk 3]
```diff
@@ -65,4 +65,4 @@
 			]
 		}
 	]
-}
+}
```
Comment (oneumyvakin) [inline fedora/solusvm2-fedora-29.json]: Please add a new line to the end of the file.

File: ubuntu/solusvm2-ubuntu-18.json [hunk 4]
```diff
@@ -84,4 +84,4 @@
       ]
     }
   ]
-}
+}
```
Comment (oneumyvakin) [inline ubuntu/solusvm2-ubuntu-18.json]: Please add a new line to the end of the file.

File: ubuntu/solusvm2-ubuntu-20.json [hunk 5]
```diff
@@ -79,4 +79,4 @@
       ]
     }
   ]
-}
+}
```
Comment (oneumyvakin) [inline ubuntu/solusvm2-ubuntu-20.json]: Please add a new line to the end of the file.

File: almalinux/solusvm2-almalinux-8-plesk.json [hunk 6]
```diff
@@ -4,23 +4,24 @@
     "password": "almalinux",
     "disk_size": "9500",
     "output_directory": "output/almalinux",
-    "output_image_name": "solus-almalinux-8-plesk"
+    "output_image_name": "solusvm2-almalinux-8-plesk"
```
Comment (oneumyvakin) [inline almalinux/solusvm2-almalinux-8-plesk.json]: Let's keep 'solus' for `output_image_name`

**Pull request merged**